docs/ProfilingModes.md

```diff
@@ -221,6 +221,40 @@ The same, when starting profiler as an agent:
 -agentpath:/path/to/libasyncProfiler.so=start,event=cpu,alloc=2m,lock=10ms,file=profile.jfr
 ```
 
+### Multi-Event Profiling with the --all Flag
+
+The `--all` parameter offers a convenient way to simultaneously enable a broad, predefined collection of common profiling events. By default, using `--all` activates profiling for `cpu`, `wall`, `alloc`, `live`, `lock` and `nativemem`.
+
+**Important Consideration**
+
+While the `--all` flag can be useful for initial exploratory analysis or development environments to get a wide overview, it may not be desirable to enable this in production (especially for continuous profiling). Users should ideally select carefully what to profile and with what settings.
```

Comment:
This is verbose, we are repeating some from above and conflicting. It is "convenient" but "should be used carefully".